pkg/controller/instance/plan_execution_engine.go

```diff
@@ -232,7 +245,7 @@ func prepareKubeResources(plan *activePlan, meta *executionMetadata, renderer ku
 	}
 
 	for _, phase := range plan.Spec.Phases {
-		phaseState, _ := getPhaseFromStatus(phase.Name, plan.State)
+		phaseState, _ := getPhaseFromStatus(phase.Name, plan.PlanStatus)
```

Comment:
check error or remove it from the return signature of getPhaseFromStatus

Reply:
I dislike the ignored error as well at first glance... however the error for an invalid name and we are getting the name for the phases.   It is hard to see that this could error unless the plans change from underneath... 

this is a private function that is use in only this use case... we should remove the error return from the function.  I would also be inclined to from the prefix `get` -> `phaseFromStatus`.    all of this is minor.